src/main/webapp/styles/addrecs.css

```diff
@@ -0,0 +1,88 @@
+/* --- Formatting for Popup Box --- */
+
+.popup .overlay {
+  background: rgba(0, 0, 0, 0.3);
+  display: none;
+  height: 100%;
+  left: 0px;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+  z-index: 0;
+}
+
+.popup .popup-content {
+  background: #fff;
+  border-color: #000;
+  border-image: initial;
+  border-style: solid;
+  border-width: initial;
+  box-sizing: border-box;
+  color: #000;
+  height: 534px;
+  left: 75%;
+  margin: 10px;
+  padding: 20px;
+  position: absolute;
+  text-align: center;
+  top: 65%;
+  transform: translate(-50%, -50%) scale(0);
+  width: 450px;
+  z-index: 2;
+}
+
+/* Close Button [X] for the Popup */
+.popup .close-btn {
```

Comment:
nit: please use `button` instead of `btn` :)

Reply:
Thanks :)